cli.js

```diff
@@ -2,13 +2,18 @@
 var Cabal = require('cabal-core')
 var swarm = require('cabal-core/swarm.js')
 var minimist = require('minimist')
+var fs = require('fs')
+var yaml = require('js-yaml')
+
 var frontend = require('./neat-screen.js')
 
 var args = minimist(process.argv.slice(2))
 
 var homedir = process.env.HOME || process.env.HOMEPATH || process.env.USERPROFILE
 var rootdir = args.dir || (homedir + '/.cabal/archives/')
```

Comment:
```suggestion
var rootdir = args.dir || (homedir + '/.cabal')
var archives = `${rootdir}/archives`
```

if we refactor like this we could save the config file in `rootdir` (this necessitates changing where `rootdir` has been used previously to `archives` instead)

Reply:
👍